app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PreviewPDFActivity.kt

```diff
@@ -51,63 +49,40 @@ class PreviewPDFActivity : AppCompatActivity(), ExternalFileInfoActionsView.OnIt
     private val navController by lazy { setupNavController() }
     private val navHostFragment by lazy { supportFragmentManager.findFragmentById(R.id.hostFragment) as NavHostFragment }
 
-    private val baseConstraintSet by lazy {
-        ConstraintSet().apply {
-            clone(binding.pdfContainer)
-        }
-    }
-    private val collapsedConstraintSet by lazy {
-        ConstraintSet().apply {
-            clone(baseConstraintSet)
-
-            clear(R.id.header, ConstraintSet.TOP)
-            connect(R.id.header, ConstraintSet.BOTTOM, ConstraintSet.PARENT_ID, ConstraintSet.TOP)
-        }
-    }
-    private val transition by lazy {
-        AutoTransition().apply {
-            duration = 125
-        }
-    }
-
     private val externalPDFUri: Uri by lazy { Uri.parse(intent.dataString) }
     private val fileNameAndSize: Pair<String, Long>? by lazy { getFileNameAndSize(externalPDFUri) }
     private val fileName: String by lazy { fileNameAndSize?.first ?: "" }
     private val fileSize: Long by lazy { fileNameAndSize?.second ?: 0 }
 
-    private var bottomSheetBehavior: BottomSheetBehavior<View>? = null
-    private var isUiShown = true
+    private val bottomSheetBehavior: BottomSheetBehavior<View> by lazy { BottomSheetBehavior.from(binding.bottomSheetFileInfos) }
```

Comment:
https://github.com/Infomaniak/android-kDrive/pull/1179#discussion_r1540869185

The reference could become wrong when only the view is recreated, I think